torch/csrc/autograd/custom_function.cpp

```diff
@@ -100,4 +100,56 @@ variable_list _wrap_outputs(const variable_list &input_vars,
   return outputs;
 }
 
+void AutogradContext::save_for_backward(const variable_list &to_save) {
+  to_save_ = to_save;
```

Comment:
nit: Since you are always going to do a copy, it is better to take `variable_list` by value. This means that if a user provides an rvalue reference, we can just move it into `to_save_`.

Reply:
(inside the body, you would then do a move into `to_save_`)